UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -213,5 +213,45 @@ public async Task WhenGetBatchInfoBasedOnProducts_ThenReturnsSearchBatchResponse
         }
 
         #endregion GetBatchInfoBasedOnProducts
+
+        #region DownloadBatchFile
+        [Test]
+        public async Task WhenGetBatchInfoBasedOnProducts_ThenDownloadBatchFilesResponse()
```

Comment:
Kindly give meaningnful name to the test case and also change the code Content = new StreamContent(new MemoryStream(Encoding.UTF8.GetBytes("Bad request"))) . It should not be bad request for valid request.

Change WhenGetBatchInfoBasedOnProducts_ThenDownloadBatchFilesResponse() to WhenGetBatchInfoBasedOnProductsReturns200_ThenDownloadBatchFiles()
Similar WhenGetBatchInfoBasedOnProductsReturnsOtherThan200_ThenDonotDownloadBatchFiles()

Reply:
updated method name